src/wallet/wallet_errors.h

```diff
@@ -302,6 +306,16 @@ namespace tools
       std::string to_string() const { return wallet_logic_error::to_string(); }
     };
 
+    struct invalid_spend_key : public wallet_logic_error
```

Comment:
Consider prepending _error to `invalid_spend_key`.

Reply:
Followed the pattern used in the rest of the file and I think it's fine. Same for other comments.